Foundation.hs

```diff
@@ -113,6 +116,12 @@ embedLayout widget = do
     pc <- widgetToPageContent widget
     withUrlRenderer $(hamletFile "templates/embed-layout-wrapper.hamlet")
 
+embedExample :: Maybe SiteId -> Widget
```

Comment:
Seems a little weird to have this in `Foundation`. Maybe just throw it in its own module?